tools/nodejs/lib/duckdb.d.ts

```diff
@@ -26,7 +26,7 @@ export class Connection {
   prepare(sql: string, ...args: [...any, Callback<Statement>] | []): Statement;
   run(sql: string, ...args: [...any, Callback<void>] | []): Statement;
 
-  register(
+  register_udf(

```

Comment:
Please add the new methods to the type stub file

Reply:
Good point! thanks